content/docs/guides/contribute/code-review.en.md

```diff
@@ -17,4 +17,23 @@ We propose you a few tips and recommendations that we think are relevant to a hu
 
 ## The code review pyramid
 
-{{< figure src="/images/docs/contribute/code_review_pyramid.svg" link="https://www.morling.dev/blog/the-code-review-pyramid/">}}
\ No newline at end of file
+{{< figure src="/images/docs/contribute/code_review_pyramid.svg" link="https://www.morling.dev/blog/the-code-review-pyramid/">}}
+
+## Script for testing a PR
+
+When reviewing a PR, it is usefull to test the changes by starting an instance of the OSRD app based on the PR branch.
+
+A script is available to spin up a separate and dedicated app instance using the PR number. The script uses the Docker images already built by the CI and launches the app, rin an isolated manner. This allows you to run both instances simultaneously without conflicts (ideal for comparing changes, for example). The key advantage is that reviews become faster since you don’t need to recompile code; you only need to download the pre-built images.
+
+Additionally, you can specify a database backup, which the script will load directly into the app.
+
+### Available Commands:
+
+* `./scripts/pr-infra-compose.sh 8914 up`: Downloads the CI-generated images for PR #8914 and launches the application.
+* `./scripts/pr-infra-compose.sh 8914 up-and-load-backup ./path_to_backup`: Downloads the images for PR #8914, restores data from the provided backup, and starts the application.
+* `./scripts/pr-infra-compose.sh 8914 down`: Shuts down the test application instance for PR #8914.
+* `./scripts/pr-infra-compose.sh 8914 down-and-clean`: Shuts down the test instance and cleans all data from the instance's docker volumes (PG data, Redis cache, RabbitMQ) to prevent all side-effects.
+
+### Accessing Services:
+
+Apart from the `frontend` server, all localhost services are available on localhost, with a minor adjustment: you’ll need to add 1 to each port number (to avoid conflicts with the dev environment) : for a list of common ports, have a lok at the [dedicated docker compose file](https://github.com/OpenRailAssociation/osrd/blob/dev/docker/docker-compose.pr-test.yml).
```

Comment:
```suggestion
Apart from the `frontend` server, all localhost services are available on localhost, with a minor port adjustment (to avoid conflicts with the dev environment): for a list of common ports, have a look at the [dedicated docker-compose file](https://github.com/OpenRailAssociation/osrd/blob/dev/docker/docker-compose.pr-test.yml).
```